### ISSUE TITLE
Accessibility Issues with Full Width Slider

### ISSUE THREAD

**Issue by clicksports** [KriesiMedia/Enfold-Feature-Requests#116]
Hello,

we noticed an accessibility issue with the buttons of the full width slider by our checking tool. The reported issue reads as this:

Guideline: [WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name](http://wcag2aa.principle4.guideline4_1.4_1_2.h91.button.name/)

Message: This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined.

Code: <button tabindex="-1"></button>

Maybe someone can check this out.
